activesupport/test/core_ext/range_ext_test.rb

```diff
@@ -60,6 +60,10 @@ def test_should_include_other_with_exclusive_end
     assert((1..10).include?(1...11))
   end
 
+  def test_include_with_endless_range
+    assert((1..).include?(2))
```

Comment:
Not sure how to handle the CI failures for this test with Ruby < `2.6.0`.

Reply:
You can wrap the whole thing around conditional:

```ruby
if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.6.0")
  def test_include_with_endless_range
    # ...
```